lib/Conversation/index.js

```diff
@@ -49,7 +50,7 @@ class Conversation extends Component {
   }
 
   render() {
-    const conversationClasses = (this.state.isActive) ? 'conversation--is-active' : '';
+    const conversationClasses = (this.state.isActive || this.props.isActive) ? 'conversation--is-active' : '';
```

Comment:
Yep that makes sense. We should handle this as a state class and not a BEM modifier 👍